app/build.gradle

```diff
@@ -4,12 +4,12 @@ plugins {
 }
 
 android {
-    compileSdk 32
+    compileSdk 33
```

Comment:
to jest ta wersja którą wybieraliśmy na początku?

Reply:
Nie, tamta wersja jest kilka linijek niżej, minSdk 26. To jest target. Na początku zrobiłem upgrada dependencies w build.gradle:app (bo mi intellij sugerował), a potem one marudziły, że chcą targetSdk ustawiony na 33

Reply:
znaczy to nie jest target, tylko compile, ale ja nie widzę pomiędzy nimi póki co różnicy

Reply:
ja pamietam ze my bralismy target najnowsza - czyli w ciagu tych dwoch tygodni nowa weszla?

Reply:
na to wychodzi